src/playermat/Playmat.ttslua

```diff
@@ -648,11 +652,40 @@ function spawnToken(position, tokenType)
   Global.call('spawnToken', {position, tokenType, PLAY_ZONE_ROTATION})
 end
 
+-- Sets this playermat's draw 1 button to visible
+---@param visibleButton Boolean.  Whether the draw 1 button should be visible
+function showDrawButton(visibleButton)
+  drawButton = visibleButton
+  if drawButton then
+    self.createButton({
+        label = "Draw 1",
+        click_function = "doDrawOne",
+        function_owner = mat,
+        position = { 1.84, 0.1, -0.36 },
+        scale = { 0.12, 0.12, 0.12 },
+        width = 800,
+        height = 280,
+        font_size = 180
+    })
+  else
+    -- remove button with index 9 if 10 buttons are present (because index starts at 0)
```

Comment:
This is going to be very fragile - e.g. the change to the larger playmats added a new discard button, which would break this.  use getButtons() and iterate it looking for the matching [something, probably label] to find the index